.github/workflows/build-matrix.org.yml

```diff
@@ -164,3 +164,39 @@ jobs:
           name: merged-content-artifact
           path: content.tar.gz
 
+  # For pull requests, deploy a preview on netlify
+  deploy-preview:
+    name: "🌐 Deploy to Netlify"
+    runs-on: ubuntu-latest
+    needs: package
+    if: ${{ github.event_name == 'pull_request' }}
+    steps:
+      - name: "📥 Content artifact download"
+        uses: actions/download-artifact@v2
+        with:
+          name: merged-content-artifact
+      - name: "📦 Extract Artifacts"
+        run: tar -xzvf content.tar.gz && rm content.tar.gz
+      - name: "📤 Deploy to Netlify"
+        id: netlify
+        # v1.2.2
+        uses: nwtgck/actions-netlify@f517512ae75beec8896aa7b027c1c72f01816200
+        with:
+          publish-dir: content
+          deploy-message: "Deploy from GitHub Actions"
+          enable-pull-request-comment: false
```

Comment:
I considered it, compared the output from both and I reckon this way looks better. Instead of a comment down in the timeline, you get a highly visible link directly at the bottom of the description. It even looks as if the PR author put it there, which makes it more seamless. And as you can see in this very PR, it will even update the link if it changed for some reason (here I updated both the prefix in CI and the site name at netlify).